docs/cli/oc4ids.rst

```diff
@@ -0,0 +1,123 @@
+* ``budget``: populates ``budget.amount`` with its equivalent
+* ``budget_approval``, ``environmental_impact``, ``land_and_settlement_impact`` and ``project_scope``: populate the ``documents`` field from ``planning.documents`` according to the ``documentType``
+* ``contracting_process_setup``: Sets up the ``contractingProcesses`` array of objects with ``id``, ``summary``, ``releases`` and ``embeddedReleases``. Some of the other transforms depend on this, so it is run first
+* ``contract_period``: populates the ``summary.contractPeriod`` field with appropriate values from ``awards`` or ``tender``
+* ``contract_price``: populates the ``summary.contractValue`` field with the sum of all ``awards.value`` fields where the currency is the same
+* ``cost_estimate``: populates the ``summary.tender.costEstimate`` field with the appropriate ``tender.value``
+* ``contract_process_description``: populates the ``summary.description`` field from appropriate values in ``contracts``, ``awards`` or ``tender``
+* ``contract_status``: populates the ``summary.status`` field using the ``contractingProcessStatus`` codelist.
+* ``contract_title``: populates ``summary.title`` from the title field in ``awards``, ``contracts`` or ``tender``
+* ``final_audit``: populate the ``documents`` field from ``contracts.implementation.documents`` according to the ``documentType``
+* ``funding_sources``: updates ``parties`` with organizations having ``funder`` in their ``roles`` or from ``planning.budgetBreakdown.sourceParty``
+* ``location``: populates the ``locations`` field with an array of location objects from ``planning.projects.locations``
+* ``procurement_process``: populates the ``.summary.tender.procurementMethod`` and ``.summary.tender.procurementMethodDetails`` fields with their equivalents from ``tender``
+* ``purpose``: populates the ``purpose`` field from ``planning.rationale``
+
+Optional transforms
+~~~~~~~~~~~~~~~~~~~
+
+Some transforms are not run automatically, but only if set. The following transforms are included if they are listed in using the ``--options`` argument (as part of a comma-separated list) or if ``--all`` is passed.
+
+* ``buyer_role``: updates the ``parties`` field with parties that have ``buyer`` in their ``roles``
+* ``description_tender``: populate the ``description`` field from ``tender.description`` if no other is available
+* ``location_from_items``: populate the ``locations`` field from ``deliveryLocation`` or ``deliveryAddress`` in ``tender.items`` if no other is available
+* ``project_scope_summary``: updates ``summary.tender`` with ``items`` and ``milestones`` from ``tender``
+* ``purpose_needs_assessment``: populate the ``documents`` field from ``planning.documents`` according to the ``documentType`` ``needsAssessment``
+* ``title_from_tender``: populate the ``title`` field from ``tender.title`` if no other is available
```

Comment:
From reading these from a user perspective, I don't have a clear idea of what happens if I don't set these.  For example, do no buyers get added to the parties array unless I set `buyer_role`? Can we specify what others might be available and take precedence?

Reply:
I think there are several areas that to docs could be clearer.  I have added this to a new issue here so it does not get lost  https://github.com/open-contracting/ocdskit/issues/134